ui/library/src/components/BoundingBox.tsx

```diff
@@ -30,6 +32,7 @@ export const BoundingBox: React.FunctionComponent<Props> = ({
   const componentClassName = classNames(
     'reader__page-overlay__bounding-box',
     isHighlighted === true ? 'reader__page-overlay__bounding-box-highlighted' : '',
+    isPopoverVisible === true ? 'selected' : '',
```

Comment:
I would use css modifier and call it something like `reader__page-overlay__bounding-box--selected` here instead of adding a `selected` class.